packages/blade/src/components/Alert/Alert.tsx

```diff
@@ -225,6 +225,7 @@ const Alert = ({
       isFullWidth={isFullWidth}
       isBorderless={isBorderless}
       {...a11yProps}
+      {...dataAttribute('blade-component', 'alert')}
```

Comment:
Yes could've just done `is-blade-component=true` but decided to anyways add names to component in case we ever need to know exactly which components are on page.

Reply:
yup this is ideal 👌